net.c

```diff
@@ -86,6 +86,16 @@ make_inet_socket(char *host, char *port)
             continue;
         }
 
+        if (host == NULL && ai->ai_family == AF_INET6) {
+            flags = 0;
+            r = setsockopt(fd, IPPROTO_IPV6, IPV6_V6ONLY, &flags, sizeof(flags));
+            if (r == -1) {
+                twarn("setting IPV6_V6ONLY off on fd %d", fd);
```

Comment:
@ysmolsky can you change this warning to "setting IPV6_V6ONLY failed ..." rather than "off", as it suggests beanstalkd has decided to disable it rather than just failing to enable it due to some external issue. Thanks

Reply:
Done, and I have corrected the commit msg and descr.